include/api/CInferenceModelDefinition.h

```diff
@@ -435,11 +436,12 @@ class API_EXPORT CInferenceModelDefinition : public CSerializableToJson {
     TTrainedModelUPtr& trainedModel();
     const TTrainedModelUPtr& trainedModel() const;
     void addToDocument(rapidjson::Value& parentObject, TRapidJsonWriter& writer) const override;
-    std::string jsonString();
+    void addToDocumentCompressed(TRapidJsonWriter& writer) const;
+    std::string jsonString() const;
+    std::stringstream jsonStringCompressedFormat() const;
```

Comment:
This file should now `#include <sstream>` - it must be included indirectly at the moment, but that could change in the future if other headers are changed.

Reply:
Fixed.